packages/stream-collector-node/src/index.ts

```diff
@@ -2,9 +2,9 @@ import { Readable } from "stream";
 import { StreamCollector } from "@aws-sdk/types";
 import { Collector } from "./collector";
 
-export const streamCollector: StreamCollector<
-  Readable
-> = function streamCollector(stream): Promise<Uint8Array> {
+export const streamCollector: StreamCollector = function streamCollector(
+  stream: Readable
+): Promise<Uint8Array> {
   return new Promise((resolve, reject) => {
     const collector = new Collector();
```

Comment:
nit: Convert to arrow function?

```typescript
export const streamCollector: StreamCollector = (
  stream: Readable
): Promise<Uint8Array> =>
  new Promise((resolve, reject) => {
    /* Code inside promise */
  });

```

Reply:
fixed in bed5267da370e90567c7969fbb18beee281c5edf